src/ingest.py

```diff
@@ -132,17 +132,20 @@ def main():
     """Ingest FITS files from a Redis queue."""
     logger.info(f"Initializing Butler from {butler_repo}")
     butler = Butler(butler_repo, writeable=True)
-    TaskClass = doImportType("lsst.obs.base.RawIngestTask")
-    ingestConfig = TaskClass.ConfigClass()
-    ingestConfig.transfer = "direct"
-    ingester = TaskClass(
-        config=ingestConfig,
+    ingest_config = RawIngestTask.ConfigClass()
+    ingest_config.transfer = "direct"
+    ingester = RawIngestTask(
```

Comment:
Ideally this clean up would be a separate commit because it's not related to the visit definitions.